src/PokemonBase.java

```diff
@@ -0,0 +1,33 @@
+public abstract class PokemonBase {
```

Comment:
"-Base" is een goede toevoeging aan de naam. 
Ook goed dat je het abstract hebt gemaakt.